misk/src/main/kotlin/misk/client/HttpClientFactory.kt

```diff
@@ -8,11 +8,25 @@ import okhttp3.OkHttpClient
 import okhttp3.Protocol
 import java.io.File
 import java.net.Proxy
+import java.time.Duration
 import java.util.concurrent.TimeUnit
 import javax.inject.Inject
 import javax.inject.Singleton
 import javax.net.ssl.X509TrustManager
 
+
+private object Defaults {
+  /*
+    Copied from okhttp3.ConnectionPool, as it does not provide "use default" option
+   */
+  val maxIdleConnections = 5
```

Comment:
I'm not sure I follow why we want these to differ from the old defaults in `HttpClientConfig`

Reply:
This is the defaults that were not present in original, as they were not nullable. I still have original values set up by always prepending `misk.client.HttpClientsConfig.Companion#getHttpClientConfigDefaults`, and this object has all the original defaults. The defaults you see here are kind of lowest-layer, in case the given HttpClientConfig somehow has nulls there (which should never happen).